analytics/request.py

```diff
@@ -2,29 +2,40 @@
 from dateutil.tz import tzutc
 import logging
 import json
-from analytics.version import VERSION
```

Comment:
I think we still need this import?

Reply:
I only moved it a few lines down :) (see below). This is for PEP8 conformance (import system modules before custom modules)